tests/cli/test_rasa_test.py

```diff
@@ -277,3 +277,59 @@ def test_test_core_help(run: Callable[..., RunResult]):
     printed_help = set(output.outlines)
     for line in lines:
         assert line in printed_help
+
+
+def test_test_core_with_entities_and_no_user_utterance(
+    run_in_simple_project_with_model: Callable[..., RunResult]
+):
+    write_yaml(
+        {"pipeline": "KeywordIntentClassifier", "policies": [{"name": "TEDPolicy"}],},
+        "config.yml",
+    )
+
+    domain_with_entity_yaml = """
+    version: "2.0"
+    intents:
+    - greet
+    entities:
+    - name
+    responses:
+     utter_greet:
+      - text: "Hey There!"
+    """
+    with open("domain.yml", "w") as domain:
+        domain.write(domain_with_entity_yaml)
+
+    nlu_yaml = """
+    version: "2.0"
+    nlu:
+    - intent: greet
+      examples: |
+      - hi [sara](name)
+      - hey
+      - hi [rasa](name)
+    """
+    with open("data/nlu.yml", "w") as nlu:
+        nlu.write(nlu_yaml)
+
+    simple_test_story_yaml = """
+stories:
+- story: happy path
+  steps:
+  - intent: greet
+    entities:
+    - name
+  - action: utter_greet
+"""
+
+    with open("data/stories.yml", "w") as stories:
+        stories.write(simple_test_story_yaml)
```

Comment:
Isn't this writing stories file under root rasa `data/stories.yml` file? How does this file get use with `run_in_simple_project_with_model`?

Reply:
Ignore me, this works. I always get confused about magic directories in tests.

Reply:
I was actually following the instructions of the bug in the issue which gets reproduced if you use the training stories as input for `rasa test core --stories`. Maybe I can make this explicit in the test by removing the writing of `tests/` and giving the training data set as a parameter to `--stories`. Let me try.